### ISSUE TITLE
Hide ok button in inline mode

### ISSUE THREAD

**Issue by davidsneighbour** [saintplay/vue-swatches#48]
When setting the swatch to inline mode I would not expect the "Ok" button to show up. 

```
            <v-swatches
                    v-model="color"
                    show-fallback
                    fallback-input-type="color"
                    popover-x="left"
                    inline
            ></v-swatches>
```

results in

![Selection_067](https://user-images.githubusercontent.com/83281/81317058-923adc00-90b6-11ea-94cd-ac6396dea016.png)

I would expect the "Ok" button to be gone, because updating the field is done without the need?

**Comment by saintplay** [comment]
yes, you are very right

**Comment by saintplay** [comment]
Fixed in [v2.0.3](https://github.com/saintplay/vue-swatches/releases/tag/v2.0.3)
